templates/collections/time_period_explorer_page.html

```diff
@@ -14,8 +14,23 @@
                 {% endfor %}
             </div>
         </div>
+        {% if FEATURE_RELATED_INSIGHTS_ON_EXPLORE_PAGES %}
+        <div class="row">
+            <div class="col-md-12">
+                 {% for insights_page in insights_time_periods %}
+                {% with page=insights_page.specific %}
```

Comment:
This `with` isn't needed, as the queryset returns instances of `InsightPage` already